roles/prometheus/defaults/main.yml

```diff
@@ -98,8 +98,8 @@ prometheus_static_targets_files:
   - prometheus/targets/*.json
 
 prometheus_scrape_config_files:
-  - prometheus/targets/*.yml
-  - prometheus/targets/*.json
+  - prometheus/scrapes/*.yml
+  - prometheus/scrapes/*.json
```

Comment:
This should probably be `scrape_configs`.

Reply:
Done. Please note that this implies a breaking change which requires manual clean-up by people who attempted to use this broken functionality.

Reply:
Could you do the cleanup or add a warning if manual cleanup is required in the preflight?

Reply:
Just deleting the folder as a clean-up is risky since people may have added their own scrape files but since it’s broken it would not have had an effect either way. What would you rather have?

Impact of not cleaning it up (automatically or manually): There is an orphaned folder with unused configuration files probably confusing administrators.

Reply:
I opted to remove the folder after all since it was never actually used but automatically created for every installation making it a hassle even for users who never attempted to use the feature to begin with.